docs/quickstart.md

```diff
@@ -64,6 +64,23 @@ Make sure that you install somewhere in your PATH.
    ark backup create nginx-backup --selector 'backup notin (ignore)'
    ```
 
+1. (Optional) Create backups based on a regular schedule using the `app=nginx` label selector:
```

Comment:
Just some wordsmithing, but I think we can simplify here. WDYT about:

2. (Optional) Set up regularly scheduled backups using a cron expression:
```
ark schedule create nginx-daily --schedule="0 1 * * *" --selector app=nginx
```

   Alternatively....

Reply:
Makes sense to me, will update.